xml-conduit/ChangeLog.md

```diff
@@ -1,3 +1,8 @@
+## 1.4.0
```

Comment:
What's the breaking change here leading to a major version bump? I had thought these were non-breaking changes.